include/alpaka/queue/QueueCpuBlocking.hpp

```diff
@@ -59,6 +60,18 @@ namespace alpaka
                     //-----------------------------------------------------------------------------
                     ~QueueCpuBlockingImpl() = default;
```

Comment:
We should now remove this destructor as it is missing the virtual keyword and it now has a base class.